Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -290,6 +290,16 @@
   categories:
     - UplinkUtility
 
+- type: listing
+  id: UplinkDimensionPot
+  name: uplink-dimension-pot-name
+  description: uplink-dimension-pot-desc
+  productEntity: DimensionPot
+  cost:
+    Telecrystal: 30
+  categories:
+  - UplinkUtility
+
```

Comment:
Blacklist this from the nukie uplink

Reply:
why though? the only possible issue with this i see is it being used as overpowered item storage, where you could just use one of the crates at nukie base just as well
alternatively, i could increase the pot's weight so it's slower to drag

Reply:
Nukies do not get this as a tool. It's just cut and dry as that. The blacklist exists for a reason.